npm/css-src/sass/_drag-and-drop.scss

```diff
@@ -2,31 +2,53 @@
   margin-bottom: 40px;
 }
 
+.drag-and-drop__button {
+  margin-bottom: 32px;
+  margin-top: 65px;
+  padding-left: 15px;
+  position: static;
+  width: auto;
+}
+
+.drag-and-drop__button::before {
+  position: static;
+}
+
+.drag-and-drop__button--highlight {
+  background-color: $govuk-focus-colour;
+  box-shadow: 0 -2px $govuk-focus-colour, 0 4px govuk-colour('black');
+  color: $govuk-text-colour;
+  outline: none;
+}
+
+.drag-and-drop--dragover {
+  background: #b1b4b6;
+  outline-color: #6f777b;
+}
+
 .drag-and-drop__dropzone {
   display: flex;
   margin-bottom: 20px;
   outline: 3px dashed $govuk-input-border-colour;
-  padding: 60px 15px;
+  padding-bottom: 30px;
+  padding-top: 0;
   text-align: center;
   transition: outline-offset 0.1s ease-in-out, background-color 0.1s linear;
 }
 
-.drag-and-drop__input {
-  display: none;
-  left: 10.1em;
-  position: relative;
-}
-
 .drag-and-drop__hint-text {
   margin-bottom: 0;
   margin-right: 10px;
-  padding-top: 7px;
+  padding-left: 15px;
+  padding-top: 70px;
 }
 
-.drag-and-drop__button {
-  display: inline-block;
-  margin-bottom: 0;
-  width: auto;
+.drag-and-drop__input {
+  height: 166px;
+  opacity: 0;
+  padding-top: 0;
+  position: absolute;
+  width: 630px;
```

Comment:
I suspect that someone looking at these rules for the first time won't be able to work out the reason for some of these styling rules. It's probably quite surprising to see an input element which has such a large height and width, and which has zero opacity. Could you add a comment explaining why we're taking this approach?

Reply:
Sure, will do!